### PR TITLE
use job_start on vim

### PR DESCRIPTION
**Pull request by mattn**

### CODE REVIEW

**Comment by mattn** [comment]
@junegunn Sorry, I can't figure out what is wrong in last one error. Can you handle this?

File: plug.vim [hunk 1]
```diff
@@ -1085,58 +1087,98 @@ function! s:update_finish()
...
-    let self.result = join(self.lines, "\n")
-    if !complete
-      call remove(self.lines, -1)
-    endif
-    " To reduce the number of buffer updates
-    let self.tick = get(self, 'tick', -1) + 1
-    if self.tick % len(s:jobs) == 0
-      call s:log(self.new ? '+' : '*', self.name, self.result)
-    endif
+    call s:job_out_cb(self, a:job_id, data)
```
Comment (mattn) [inline plug.vim]: right. thanks.

**Comment by junegunn** [comment]
Cool, I'll take a look later in the day. Note that vader.vim, the test framework we use here, is currently unable to test asynchronous installer, so we can see that the travis builds are not running correctly except for the first one where Vim 7.2 is used (`ENV=nox`). I'll also check if I can extend vader to support async features.

**Comment by junegunn** [comment]
@mattn Sorry for the delay. I decided that we need to set up proper automated testing for async installers. We already have Neovim installer that is not at all covered by the tests and I don't feel good about adding another one that is also not tested. It means that I have to extend vader.vim for asynchronous behaviors. I have some ideas and I'll let you know how it turns out.

**Comment by mattn** [comment]
Sure. let me known if the time come.

**Comment by junegunn** [comment]
Alright, I think I got it working. All the tests are now passing.
https://travis-ci.org/junegunn/vim-plug/builds/168217454

I made a few changes to your patch. Can you review the commits in https://github.com/junegunn/vim-plug/commits/vim8 ?

**Comment by mattn** [comment]
Working good for me!

vim-plug with job on vim8

**Comment by junegunn** [comment]
Thanks, good to see that it's working correctly on Windows as well. I'll review the changes again and merge it to master soon.

**Pull request closed without merging**